roles/rclone_serve_restic/defaults/container.yml

```diff
@@ -0,0 +1,58 @@
+---
+
+rclone_serve_restic_container_image_reference: >-
+  {{
+    rclone_serve_restic_container_image_repository
+    ~ ':'
+    ~ rclone_serve_restic_container_image_tag | default(rclone_serve_restic_version)
+  }}
+
+rclone_serve_restic_container_image_repository: >-
+  {{
+    (
+      container_registries[rclone_serve_restic_container_image_registry]
+      | default(rclone_serve_restic_container_image_registry)
+    )
+    ~ '/'
+    ~ rclone_serve_restic_container_image_namespace | default('')
+    ~ rclone_serve_restic_container_image_name
+  }}
+
+rclone_serve_restic_container_image_registry: "docker.io"
+rclone_serve_restic_container_image_namespace: "rclone"
```

Comment:
this will result in `docker.io/rclonerclone`, as either `rclone_serve_restic_container_image_namespace` needs to end with a slash or L17 needs to insert a slash if the variable is a non-empty string.

Reply:
might work like this, needs to be tested before merged